src/legacy/ui/public/styles/_legacy/components/_ace_overrides.scss

```diff
@@ -1,205 +0,0 @@
-@import '@elastic/eui/src/components/call_out/variables';
-@import '@elastic/eui/src/components/call_out/mixins';
-
-// SASSTODO: Replace with an EUI editor
-// Intentionally not using the EuiCodeBlock colors here because they actually change
-// hue from light to dark theme. So some colors would change while others wouldn't.
-// Seemed weird, so just hexing all the colors but using the `makeHighContrastColor()`
-// function to ensure accessible contrast.
-
-// In order to override the TM (Textmate) theme of Ace/Brace, everywhere,
-// it is being scoped by a known outer selector
-.application {
-  .ace-tm {
-    $aceBackground: tintOrShade($euiColorLightShade, 50%, 0);
```

Comment:
These styles are definitely used. They match up Ace/Brace editor themes to EUI. Do not delete.

Reply:
ok, thx will restore! and will move them to `src/legacy/ui/public/styles/_base.scss` if this is confirmed